core/issuers.js

```diff
@@ -122,6 +122,10 @@ Description must be truthy: ${description}`;
       return assay;
     },
 
+    getStrategy() {
```

Comment:
Does anyone call this? Do you expect anyone to?

Reply:
Nothing calls it right now, but my state machine version of scooter plans to make use of this